omise-wp-admin.php

```diff
@@ -51,6 +51,7 @@ public function register_admin_page_and_actions() {
 		public function add_dashboard_omise_menu() {
 			add_menu_page( 'Omise', 'Omise', 'manage_options', 'omise-plugin-admin-page', array( $this, 'init_dashboard' ) );
 			add_submenu_page( 'omise-plugin-admin-page', 'Omise Dashboard', 'Dashboard', 'manage_options', 'omise-plugin-admin-page' );
+			add_submenu_page( 'omise-plugin-admin-page', 'Omise Transfers', 'Transfers', 'manage_options', 'omise-plugin-admin-transfer-page', array( $this, 'init_transfers' ) );
```

Comment:
@nimid what do you think if we will rename `init_dashboard` & `init_transfer` methods to something that more understandable that that methods are for render view. Maybe, 
- `init_transfer_page()`, `init_dashboard_page()` 
- `transfer_page()`, `dashboard_page()` or 
- `page_transfer()`, `page_dashboard()`